FirebaseAI/Sources/GenerativeAIService.swift

```diff
@@ -207,6 +207,29 @@ struct GenerativeAIService {
     return urlRequest
   }
 
+  private func requestURL<T: GenerativeAIRequest>(request: T) throws -> URL {
+    guard var urlComponents = URLComponents(url: request.url, resolvingAgainstBaseURL: false) else {
+      throw URLError(.badURL, userInfo: [
+        NSLocalizedDescriptionKey: "Invalid Request URL: \(request.url)",
+      ])
+    }
+    var urlQueryItems = urlComponents.queryItems ?? []
+
+    // The query parameter `$outputDefaults` forces the backend to output proto default values for
+    // JSON responses instead of omitting them. See
+    // https://cloud.google.com/apis/docs/system-parameters#definitions for more details.
+    urlQueryItems.append(URLQueryItem(name: "$outputDefaults", value: "true"))
```

Comment:
Will this increase cost?

Reply:
Only in terms of bandwidth since the JSON payload will be larger in cases where defaults were previously omitted.

Note: Google AI actually behaves this way by default (default values have always been included in the JSON). Vertex AI always omits them unless this option is specified.

This has also caused some confusion in the past since, for example, `blocked: false` was omitted in Vertex since `false` is the default. Other cases tend to revolve around `0` getting omitted for numeric fields.